ansible/roles/projects-common/deploy-sriov-device-plugin/templates/sriov-dp-config-map.yaml.j2

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriovdp-config
+  namespace: kube-system
+data:
+  config.json: |
+    {
+        "resourceList": [{
+                "resourcePrefix": "{{ device_plugin_prefix }}",
+                "resourceName": "{{ device_plugin_resource }}",
+                "selectors": {
+                    "vendors": ["15b3"],
+                    "devices": ["1018"],
```

Comment:
Since we use pfNames, can you remove the device id. Since we dont want to restrict it to connectX5

Reply:
fixed